Rails 5

jhawthorn opened the pull request.

jhawthorn committed:
Remove deprecated raise_in_transactional_callbacks

jhawthorn committed:
Ensure players is array before calling combination

jhawthorn committed:
Change format of controller specs

jhawthorn committed:
Attempt to upgrade to rails 5

jarednorman commented on spec/controllers/api/ongoing_matches_controller_spec.rb:
```diff
@@ -188,7 +188,7 @@
 
   describe "DELETE destroy" do
-    subject { delete :destroy, { format: :json }, { write_access: true } }
+    subject { delete :destroy, format: { format: :json }, session: { write_access: true } }
```
Did you mean `params`? You did this a few times.

jhawthorn committed:
Use bang methods in sample data

jhawthorn committed:
Force use of inline queue adapter

This used to be the default but was replaced with :async in rails 5. We
want to continue using inline for now as our specs and setup depend on
it.

jhawthorn committed:
Fix deprecations in controller specs

jhawthorn committed:
Use redirect_back instead of redirect_to :back

jhawthorn committed:
Use present? && instead of try!

try! doesn't seem to be playing nicely with a SimpleDelegator wrapping
nil.

jhawthorn committed:
Require dependency PingPong::Service

We lock up without this.... no idea why

jhawthorn committed:
Introduce serialAjax for bacon

We want to send all our alax calls from the frontend in a serialized
manner to avoid parallelism issues. Previously it was possible for
events to arrive out of order.

jhawthorn committed:
Remove fixed username from database.yml

This was added to simplify the elixir split and should no longer be
needed.

jhawthorn commented:
:shipit:
